gyro/lib/TaskManager.js

```diff
@@ -368,11 +370,20 @@ TaskManager.prototype._runTask = function(taskName) {
     // actually spawn the process
     var startTime = Date.now();
     var self = this;
-    this._runningProcesses++;
+
+    /* Some tasks spawn their own processes. Use config.processMultiplier
+        to represent the number of processes a task might spawn. */
+    var taskConfig = task.getConfig() || {};
+    var processMultiplier = parseInt(taskConfig['processMultiplier'] || 1);
```

Comment:
I'd probably name this something like `processesUsed` or something like that given that it's not being used as a multiplier here